packages/grafana-toolkit/src/cli/tasks/plugin.dev.ts

```diff
@@ -2,11 +2,36 @@ import { Task, TaskRunner } from './task';
 import { bundlePlugin as bundleFn, PluginBundleOptions } from './plugin/bundle';
 import { useSpinner } from '../utils/useSpinner';
 
+// @ts-ignore
+import execa = require('execa');
+import path = require('path');
+
 const bundlePlugin = useSpinner<PluginBundleOptions>('Bundling plugin in dev mode', options => {
   return bundleFn(options);
 });
 
+const yarnlink = useSpinner<void>('Linking local toolkit', async () => {
+  // await execa('yarn', ['remove', '@grafana/toolkit']);
+
+  // Add all the same dependencies as toolkit
+  let args: string[] = ['add'];
+  const packages = require(path.resolve(__dirname, '../../../package.json'));
+  for (const [key, value] of Object.entries(packages.dependencies)) {
+    args.push(`${key}@${value}`);
+  }
+  await execa('yarn', args);
+  await execa('yarn', ['link', '@grafana/toolkit']);
+
```

Comment:
@dprokop -- maybe crazy, but thinking more about enzyme.  It would be great to have that as our standard out-of-the-box best practice.

Could we do an `ln -s ???` in this script to make sure the plugin and toolkit are using the same instance?

Reply:
Not sure, need check it out. I don't think this should be prioritised now though. Let's have the whole pipeline working, and then we can extend on supporting std libs, shall we?

Reply:
agree -- lets put it on the list, I'm just pointing out it went from "not very likely", to "maybe an easy solution" since the problem is just for dev and works fine in plugins.